packages/common/package.json

```diff
@@ -12,33 +12,33 @@
     "./dist/*"
   ],
   "exports": {
-    "./components/*": "./dist/src/components/*",
-    "./hooks/*": "./dist/src/hooks/*",
-    "./polyfills/*": "./dist/src/polyfills/*",
-    "./utils/*": "./dist/src/utils/*"
+    "./components/*": "./dist/components/*",
+    "./hooks/*": "./dist/hooks/*",
+    "./polyfills/*": "./dist/polyfills/*",
+    "./utils/*": "./dist/utils/*"
   },
   "typesVersions": {
     "*": {
       "components/*": [
-        "dist/src/components/*"
+        "dist/components/*"
       ],
       "hooks/*": [
-        "dist/src/hooks/*"
+        "dist/hooks/*"
       ],
       "polyfills/*": [
-        "dist/src/polyfills/*"
+        "dist/polyfills/*"
       ],
       "utils/*": [
-        "dist/src/utils/*"
+        "dist/utils/*"
       ]
     }
   },
   "scripts": {
     "clean": "rimraf ./dist ./coverage tsconfig.tsbuildinfo",
     "clean:all": "npm run clean -- ./node_modules",
     "build": "npm run clean && npm run compile && npm run copy:css",
-    "compile": "tsc --build --verbose",
-    "copy:css": "copyfiles -u 1 ./src/**/*.css ./dist/src",
+    "compile": "tsc --build --verbose && tsc-alias -p tsconfig.json",
```

Comment:
It gets tricky to do both a `tsc --build` and a `tsc-alias` and allow editing of files in common while the webpack dev server is running.

When webpack dev server is running with project references turned on, `tsc --build` on the referenced projects is implied.  So if you change a common package file, it will transpile properly, but the aliases won't be replaced.

It is quite a PIA.

Reply:
see next comment about paths in plugin bypassing this mechanism in dev mode
e.g 
in dev mode the pluin see only the source tree